package.json

```diff
@@ -158,7 +159,7 @@
   ],
   "lint-staged": {
     "**/*.{js,jsx,tsx,ts,json}": [
-      "eslint --fix"
+      "pnpm run lint:fix"
```

Comment:
Why add `pnpm run lint:fix`?

Reply:
`eslint --fix` will execute lint for the files changed in git stage.And `pnpm run lint:fix` will run the complete process for all files.

Reply:
Sorry, I just see this [article](https://juejin.cn/book/7050063811973218341/section/7058853948060336163) uses the `npm run lint:script` in lint-staged.